proto/cosmos/gov/v1beta1/gov.proto

```diff
@@ -8,10 +8,7 @@ import "google/protobuf/timestamp.proto";
 import "google/protobuf/any.proto";
 import "google/protobuf/duration.proto";
 
-option go_package                       = "github.com/cosmos/cosmos-sdk/x/gov/types";
-option (gogoproto.goproto_stringer_all) = false;
-option (gogoproto.stringer_all)         = false;
-option (gogoproto.goproto_getters_all)  = false;
```

Comment:
The approach I took here is: let the codegen generate as much code as possible, so that we don't write extra manual code ourselves.

In v0.42: `stringer_all=false`, and we added our custom implementation of `String()`
Now: `stringer_all=true`, and we use the codegen's implementation of `String()` (so less custom implementation)

Migration code anyways never call this `String()` method. But if it's confusing I can revert back to exactly what was in 0.42.

Reply:
this is gogo specific, not part of the proto definition. So we are safe to remove it keeping backward compatibility.

Reply:
Thanks for the explanation. No need to revert back to exactly what was in 0.42.